### DISCUSSION TITLE
Zoo models usage

### DISCUSSION THREAD

**Discussion by SNESEAR** [deepjavalibrary/djl#3557] [Development]
Hi, I'm trying to start using djl, and I can't find any information about zoo models params.

So I tried to run a test example from the documentation (face recognition with drawing rectangles), everything worked, but I want to recognize a face to draw a face rectangle, also recognize a face from 2 different images, as it is in another example from the documentation, and also recognize face landmarks to calculate (using opencv) head pose estimation, or maybe there is a model that returns it. How to find a model, that can detect faces, give encodings (face unique vector), landmarks (or head pose estimation).

 Where to find information about required parameters?

**Comment by frankfliu** [comment]
@SNESEAR 

You talked about 3 different tasks (or model):
1. face detection (including landmark), this is included in the example:  https://github.com/deepjavalibrary/djl/blob/master/examples/src/main/java/ai/djl/examples/inference/face/FaceDetectionTranslator.java
2. face recognition, we actually don't have such model in our model zoo, it's considered as a subset of image classification. In our example, it's actually a image feature extraction model, then you can compare image similarity.
3. pose estimation, but we only has [yolo pose estimate](https://docs.ultralytics.com/tasks/pose/), which for may not work for your use case.

DJL model zoo is mainly used for show case what DJL is capable of. It's not meant to collect all models. See [this](https://docs.djl.ai/master/docs/load_model.html#list-available-models-using-djl-command-line) for how to get a complete list of models in model zoo.

You may want to search on HuggingFace for models, and then convert it to DJL to run in Java. For some of models you may also need to implement your own `Translator` to make it work.

**Reply by SNESEAR** [reply]
thanks
